addons/im_livechat/models/im_livechat_channel_member_history.py

```diff
@@ -191,3 +192,11 @@ def action_open_discuss_channel_view(self, domain=()):
         action["view_mode"] = "list"
         action["views"] = [view for view in action["views"] if view[1] in ("list", "form")]
         return action
+
+    def _to_store_defaults(self, target):
+        return [
+            Store.One("channel_id", [], as_thread=True),
+            Store.One("guest_id", ["name"]),
+            "livechat_member_type",
+            Store.One("partner_id", self.env["res.partner"]._get_store_livechat_username_fields()),
```

Comment:
`guest_id` and `partner_id` are mutually exclusive so we could optimize by only sending one when the other is not set (both can be null so we cannot check the presence of the field itself)